VespaGAS.xcodeproj/project.pbxproj

```diff
@@ -8,6 +8,7 @@
 
```

Comment:
I think what @dominicfreeston means is that just providing the `==` function isn't sufficient; you need to explicitly say that a type conforms to `Equatable`, i.e., you also need to add `: Equatable` to the type declaration.